src/test/test_sign.py

```diff
@@ -377,5 +382,49 @@ def add_by_final_vbf(a_hex, b_hex):
         self.assertEqual(mul(zero, scalar), (WALLY_OK, zero_hex))
         self.assertEqual(mul(zero, negative), (WALLY_OK, zero_hex))
 
+    def test_bip340_sigs(self):
+
+        made_sig = False
+
+        for case in self.get_bip340_sign_cases():
+            priv_key = make_cbuffer(case["priv_key"])
```

Comment:
I prefer e.g. `priv_key, priv_key_len = make_cbuffer(case['priv_key'])` and then remove the `[0]` and `[1]` subscripting below.

Please note the use of `'` instead of `"` for strings (except autodoc lines) in the python code, this for for ease of grepping.

Reply:
done